### PR TITLE
Add inference results to videocomposer doc

### PR DESCRIPTION
**Pull request by HaoyangLee**

### CODE REVIEW

**Comment by zhtmike** [comment]
is it possible to make the GIF keep playing (not only play once) when browsing?

File: examples/videocomposer/README.md [hunk 1]
```diff
@@ -2,6 +2,129 @@
...
+
+<br />
+<div align="center">
+<img src="https://github.com/HaoyangLee/mindone/assets/20376974/cdaabe13-3d28-4a25-bc00-1dc86e6507f3" width="720" />
+</div>
+<p align="center">
+<em> Condition: sketch <br /> Text input: "A black swan swam in the water" </em>
+</p>
+
+### Motion Transfer from a video to a Single Image
```
Comment (wtomin) [inline examples/videocomposer/README.md:48]: The title capitalization can be improved, e.g., Motion Transfer from a video to a Single Image -> Motion Transfer From a Video to a Single Image. Pls check some other titles using https://capitalizemytitle.com

**Comment by wtomin** [comment]
> is it possible to make the GIF keep playing (not only play once) when browsing?

According to this [QA](https://stackoverflow.com/questions/72508378/enable-gif-autoplay-on-github-readme), it is related to Giithub [Accessibility Settings](https://github.com/settings/accessibility).

**Comment by zhtmike** [comment]
> > is it possible to make the GIF keep playing (not only play once) when browsing?
> 
> According to this [QA](https://stackoverflow.com/questions/72508378/enable-gif-autoplay-on-github-readme), it is related to Giithub [Accessibility Settings](https://github.com/settings/accessibility).

I mean if it is possible to play many times (as a infinite loop), not only once

File: examples/videocomposer/README.md [hunk 2]
```diff
@@ -2,6 +2,138 @@
 
 MindSpore implementation & optimization of [VideoComposer: Compositional Video Synthesis with Motion Controllability](https://arxiv.org/pdf/2306.02018.pdf).
 
+## Gallery
+
+### Inference different conditions from a video
+<p align="center">
+<img src=assets/y-depth_vid0003.gif width="720" />
```
Comment (SamitHuang) [inline examples/videocomposer/README.md:9]: why not upload as other gifs using git content url

**Comment by SamitHuang** [comment]
> is it possible to make the GIF keep playing (not only play once) when browsing?

@HaoyangLee You may re-use the `save_videos` API here, which supports endless playing gif
https://github.com/mindspore-lab/mindone/blob/6657b3b8b031b64f7062773a277d8541712d0f98/mindone/visualize/videos.py#L7

File: examples/videocomposer/README.md (hunk 2 above)
Comment (HaoyangLee) [reply examples/videocomposer/README.md:9]: It's a draft. Still trying different ways to make the gif loop infinitely. Will use the git content url.

**Comment by HaoyangLee** [comment]
The gifs can loop infinitely now.

File: examples/videocomposer/README.md (hunk 1 above)
Comment (HaoyangLee) [reply examples/videocomposer/README.md]: fixed

**Review by zhtmike — APPROVED** [review]

**Comment by wtomin** [comment]
> The gifs can loop infinitely now.

Nice job! What is the trick to make it loop?

**Review by wtomin — APPROVED** [review]

**Comment by HaoyangLee** [comment]
> > The gifs can loop infinitely now.
> 
> Nice job! What is the trick to make it loop?

Two ways:
1. Gif editing tools such as https://ezgif.com/maker or photoshop can change the number of loop easily.
2. Use PIL api (ref. [(1)](https://github.com/python-pillow/Pillow/issues/4575) [(2)](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#gif-saving)) or [mindone api](https://github.com/mindspore-lab/mindone/blob/6657b3b8b031b64f7062773a277d8541712d0f98/mindone/visualize/videos.py#L7).

File: examples/videocomposer/README.md (hunk 2 above)
Comment (hadipash) [reply examples/videocomposer/README.md]: There's no issue with playing a GIF indefinitely on GitHub:
https://github.com/mindspore-lab/mindone/blob/master/examples/svd/README.md
When you save the GIF, you need to specify the `loop` parameter for it to play indefinitely.

**Review by hadipash — APPROVED** [review]

**Pull request merged**